Doc/library/csv.rst

```diff
@@ -156,8 +156,10 @@ The :mod:`csv` module defines the following classes:
 
    The *fieldnames* parameter is a :term:`sequence`.  If *fieldnames* is
    omitted, the values in the first row of file *f* will be used as the
-   fieldnames.  Regardless of how the fieldnames are determined, the
-   dictionary preserves their original ordering.
+   fieldnames and the first row will not produce a dict in the result.  If
```

Comment:
Since the first row will be used as the fieldnames and be omitted, how about writing this line as simply:

> "...and therefore be omitted. If..."